### ISSUE TITLE
Empty struct generated from interface description

### ISSUE THREAD

**Issue by lostwald** [Yakindu/statecharts#1188]
Hi,

the following interface description generates a empty struct:

![interface](https://cloud.githubusercontent.com/assets/23447929/22649556/bbccd95c-ec7b-11e6-9d37-688d22733a40.png)

![struct](https://cloud.githubusercontent.com/assets/23447929/22649583/dacdbcea-ec7b-11e6-9879-46bc5c8ee2c1.png)

**Comment by terfloth** [comment]
Seems to be identical to of #596. This has to be qualified.

**Comment by BeckmaR** [comment]
Hey @lostwald, this should have been fixed in release 2.7.1, according to the commit history (#869 merged on August 3rd 2016). Could you please verify that the version used here was not older than that?

**Comment by BeckmaR** [comment]
I can verify this issue still exists.

This statechart:
![bildschirmfoto vom 2017-02-08 09-44-32](https://cloud.githubusercontent.com/assets/15799654/22729484/3a7b9546-ede3-11e6-9653-79a2cd2f1985.png)
produces this empty interface struct in `OSM.h`:
```
/*! Type definition of the data structure for the OSMInternal interface scope. */
typedef struct
{
} OSMInternal;
```
Instead, the following line is added to `OSM.c`:
```
const sc_integer OSM_OSMINTERNAL_OSM_STATE_INIT = 0;
```
The code seems to be referencing this constant correctly. A getter-function is not generated.

If the `const` is moved to an interface, the same happens - an empty struct, a const in `OSM.c`, and a getter function is generated. Trying to access the consts directly obviously does not work, because the symbol is not known.

Basically, it could be worse, everything works, only the code style is bad.
